composer.json

```diff
@@ -71,9 +74,11 @@
 		"silverorange/xml_rpc2": "Enables XML-RPC server pages."
```

Comment:
Similarly this is required for some features. It probably should just be a require.

Reply:
Moved `xml_rpc_ajax` and `xml_rpc2` into the `require` section.